PowerEditor/src/Parameters.cpp

```diff
@@ -2792,6 +2792,16 @@ void NppParameters::writeUserDefinedLang()
 
 void NppParameters::insertCmd(TiXmlNode *shortcutsRoot, const CommandShortcut & cmd)
 {
+	//insert a comment about which command is written in the XML file
+	TiXmlNode *sc2 = shortcutsRoot->InsertEndChild(TiXmlComment());
+	generic_string str = TEXT("");

```

Comment:
Why the explicit ` = TEXT("")`? `std::wstring` already default constructs.

Explicitly initializing it adds (a *really* tiny amount of) code bloat.